bson/src/main/org/bson/codecs/pojo/Conventions.java

```diff
@@ -69,8 +69,17 @@ public final class Conventions {
      */
     public static final Convention USE_GETTERS_FOR_SETTERS = new ConventionUseGettersAsSettersImpl();
 
-
     /**
+     * A convention that <em>only uses fields</em> for setting and getting values to be serialized. Automatically,
```

Comment:
Let's also document whether this convention will encode/decode fields that _don't_ have a corresponding getter and/or setter.  I'm not sure whether it should or not, actually.

Reply:
Its documented as only using fields - so wont use any methods for encoding / decoding.

Reply:
There are two pieces to it:

1. Choosing which properties to include, based on existence of field/getter/setter
2. Once properties are included/excluded, choosing how to get/set the value

The second is clear to me, but the first is not.

Reply:
Ack - will clarify

Reply:
Added some text to clarify.

Reply:
I'm still finding this hard to interpret. Perhaps we can wordsmith it together in person..